tests/test_auth.py

```diff
@@ -42,6 +42,13 @@
 
 class TestAuthUtils(OsfTestCase):
```

Comment:
Especially since I'm asking for several more tests, it would be best not to use a test case that initializes the database for these tests.